static/js/index.js

```diff
@@ -216,6 +216,12 @@
   var renderHits = function(search, result) {
 
     var html = "";
+
+    if (result.ResultCount) {
```

Comment:
Updated. Sorry about that. First time Go user here :)

On Tue, Mar 15, 2016 at 9:40 PM, Sylvain Zimmer <notifications@github.com>
wrote:

> In static/js/index.js
> <https://github.com/commonsearch/cosr-front/pull/29#discussion_r56280973>:
>
> > @@ -216,6 +216,12 @@
> >    var renderHits = function(search, result) {
> >
> >      var html = "";
> > +
> > +    if (result.ResultCount) {
>
> This should be result["c"] now? (and not result.c because of the closure
> compiler optimizations!)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/commonsearch/cosr-front/pull/29/files/42ffa509ec792b1970e47adaa04a209f3d31208c#r56280973>
>



-- 
Yu Wu